test/facility-locator/reducers/searchQuery.unit.spec.js

```diff
@@ -0,0 +1,88 @@
+import { expect } from 'chai';
+
+import searchQueryReducer from '../../../src/js/facility-locator/reducers/searchQuery';
+
+const INITIAL_STATE = {
+  searchString: '',
+  serviceType: null,
+  facilityType: null,
+  position: {
+    latitude: 38.8976763,
+    longitude: -77.03653,
+  },
+  bounds: [
+    -77.53653,
+    38.3976763,
+    -76.53653,
+    39.3976763,
+  ],
+  context: 20004,
+  inProgress: false,
+  currentPage: 1,
+  zoomLevel: 11,
+  searchBoundsInProgress: false,
+};
+
+describe('search query reducer', () => {
+  it('should handle search started', () => {
+    const state = searchQueryReducer(INITIAL_STATE, {
+      type: 'SEARCH_STARTED',
+    });
+
+    expect(state.error).to.eql(false);
+    expect(state.inProgress).to.eql(true);
+  });
+
+  it('should handle fetching list of facilities', () => {
+    const state = searchQueryReducer({
+      ...INITIAL_STATE,
```

Comment:
`INITIAL_STATE` is destructured here but not in the subsequent tests. Either add it in later tests or remove this one for consistency?